user_docs/en/changes.t2t

```diff
@@ -68,7 +69,7 @@ What's New in NVDA
 
 == Changes for Developers ==
 - Note: this is a Add-on API compatibility breaking release. Add-ons will need to be re-tested and have their manifest updated.

```

Comment:
"a add-on" s/b "an add-on", and "manifest" s/b "manifests". Also, should the first "Add-on" be capitalized?

```suggestion
- Note: this is an add-on API compatibility breaking release. Add-ons will need to be re-tested and have their manifests updated.

```